arrows/core/associate_detections_to_tracks_threshold.h

```diff
@@ -48,10 +48,9 @@ class KWIVER_ALGO_CORE_EXPORT associate_detections_to_tracks_threshold
 {
 public:
 
-  static constexpr char const* name = "threshold";
-
-  static constexpr char const* description =
-    "Associate detections to tracks via simple thresholding on the input matrix.";
+  PLUGIN_INFO( "threshold",
+               "1.0",
```

Comment:
Do we really need/want version numbers on each algorithm?  I can't see anyone every changing these in a meaningful way.  I think it makes more sense to have a version number for the whole plugin (which may show up in the explorer when you inspect a given algorithm).  The version numbers for the plugins in KWIVER would probably just be the same as the version number of KWIVER.  External plugins would have to manage there own version numbers.  Does that make sense?   Or is there a need for more fine-grained versioning?